core-web/libs/sdk/uve/src/lib/types/editor/internal.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @description Custom client parameters for fetching data.
+ */
+export type DotCMSClientParams = {
```

Comment:
To avoid confusion with `@dotcms/client` 

```suggestion
export type DotCMSCustomerParams = {
```